analytical_engine/core/grape_engine.cc

```diff
@@ -146,9 +147,29 @@ extern "C" void master_signal_handler(int sig, siginfo_t* info, void* context) {
   }
 }
 
+// A custom sinker to redirect glog messages to stdout/stderr.
+class RedirectLogSink : public google::LogSink {
```

Comment:
Are you sure we need a `redirectlogsink`? I think glog can be configured to send error/out to stdout/stderr?

Reply:
glog can only set to send log to stderr, others would write to log file.
FYI: https://github.com/google/glog#setting-flags